src/cljs/rems/navbar.cljs

```diff
@@ -34,16 +41,30 @@
                   (str/starts-with? location path))]
     [nav-link-impl path title active?]))
 
+
+(defn nav-link-options
+  "A link to path that is shown as active when the current browser location matches the path.
+
+   By default checks if path is a prefix of location, but if match-mode is :exact,
+   checks that path is exactly location."
+  [opts path title & [match-mode]]
+  (let [location @(rf/subscribe [:path])
+        active? (case match-mode
+                  :exact
+                  (= location path)
+                  ;; default: prefix
+                  (str/starts-with? location path))]
+    [nav-link-impl-opts opts path title active?]))
+
 (defn user-widget [user]
   (when user
     [:div.user-widget.px-2.px-sm-0
-     [:span
-      [:i.fa.fa-user.mr-1]
-      [:span.user-name (:name user)]]
-     [atoms/link {:id "settings" :class "nav-link"} (url-dest "/profile")
+     [nav-link-options
```

Comment:
I think you can just use the normal `nav-link` here if you give up on the `:id` for this link. I think no id is fine, since we don't have ids for our other nav links either.

Alternatively, if you really want to keep the id, you can add an additional optional argument to `nav-link`.

Reply:
I added it without id (since no real need now)